README.md

```diff
@@ -28,81 +37,138 @@ $ source ea-venv/bin/activate
 
-## Development
+`$ encord-active import project`
 
-### Write your own metrics
+Note, there are also options for importing projects from, e.g,. KITTI, CVAT, and COCO. 
+Find more details in [the documentation][encord-active-docs-workflow-import-data].
 
-Encord Active isn't limited to the metrics we provided, it is actually quite easy to write your own 🔧
-See the [Writing Your Own Metric](https://docs.encord.com/admins/settings/public-keys/#set-up-public-key-authentication) page on the WIKI for details on this.
 
-### Pre-commit hooks
+## ⭐ Concepts and features
 
-If you have installed the dependencies with poetry, then you can install pre-commit hooks by running the following command:
+### 📊 Quality metric
 
-```shell
-$ pre-commit install
-```
+Quality metric are applied to your data, labels, and predictions to assign them quality metric scores. 
+Plug in your own or rely on Encord Actives prebuilt quality metrics. 
+The quality metrics automatically decompose your data, label, and model quality to show you how to improve your model performance from a data-centric perspective. 
+Encord Active ships with 25+ metrics and more are coming; [contributions][contribute-url] are also very welcome.
 
-The effect of this will be that `black`, `isort`, `mypy`, and `pylint` needs to run without finding issues with the code before you are allowed to commit.
-When you commit and either `black` or `isort` fails, committing again is enough, as the side effect of committing the first time is to reformat files.
+**Other core features:**
 
-Running each of the tools individually on your code can be done as follows:
+- [Model Decomposition](https://encord-active-docs.web.app/pages/model-assertions/metrics)
+- [Data Exploration](https://encord-active-docs.web.app/pages/data-quality/summary)
+- [Similarity Search](https://encord-active-docs.web.app/workflows/Improve-your-data/similar-images)
+- [Annotator Benchmarks](https://encord-active-docs.web.app/pages/label-quality/explorer/)
+- [Data Tagging](https://encord-active-docs.web.app/tags/#steps)
+- [COCO Exports](https://encord-active-docs.web.app/pages/export/filter_export)
+- And much more!
 
-```shell
-$ poetry run black --config=pyproject.toml .
-$ poetry run isort --sp=pyproject.toml .
-$ poetry run mypy . --ignore-missing-imports
-$ poetry run pylint -j 0 --rcfile pyproject.toml [subdir]
-```
+Visit our [documentaiton][encord-active-docs] to learn more.
 
-## Community and Support
+### Supported data:
 
-Join our community on [Slack][join-slack] and [Twitter][twitter-url]!
+| Data |  | Labels |  | Project sizes |  |
+| --- | --- | --- | --- | --- | --- |
+| `jpg` | ✅ | Bounding Boxes | ✅ | Images | 100.000 |
+| `png` | ✅ | Polygons | ✅ | Videos | 100.000 frames |
+| `tiff` | ✅ | Segmentation | ✅ |  |  |
+| `mp4` | ✅ | Classifications | 🟡 |  |  |
+|  |  | Polylines | 🟡 |  |  |
 
-[Suggest improvements and report problems][new-issue]
+## 🧑🏽‍💻Development
 
-# Contributions
+### 🛠 Write Your Own Metrics
```

Comment:
[nitpick] 💅

Seems that most headlines are capitalizing only the first character of the first word. should we perhaps be consistent?

```suggestion
### 🛠 Write your own metrics
```